### ISSUE TITLE
Title encoding

### ISSUE THREAD

**Issue by Fxbouffant** [NonFactors/MVC6.Grid#35]
When using a special character in a column title `.Titled("é")`, it is not correctly displayed even though the content of the grid is correctly displaying special characters.

**Comment by Muchiachio** [comment]
Hmm, I'm not able to reproduce, could you share your environment
- ASP NET version
- MVC Grid version
- Browser

Some time ago I did change `Titled` method to use title string as raw html, to enable rich header. Maybe that's causing it.

**Comment by Fxbouffant** [comment]
ASP.NET Core version 1.1.0
MVC Grid Version - 1.3.0

With Google Chrome 54
![image](https://cloud.githubusercontent.com/assets/3736111/20596779/58177dba-b240-11e6-90ad-8ae706eb9ea3.png)

With IE 11
![image](https://cloud.githubusercontent.com/assets/3736111/20596787/6415a2a4-b240-11e6-957e-64f513f0ce54.png)


I'm creating a project like this:
![image](https://cloud.githubusercontent.com/assets/3736111/20596589/922312a4-b23f-11e6-949f-fe02d2914507.png)

**Comment by Muchiachio** [comment]
I created a small project [here](https://github.com/Muchiachio/HeaderTest), could you check and see if it displays incorrectly for you. I get this now
![image](https://cloud.githubusercontent.com/assets/6647444/20600044/478f19e8-b25a-11e6-81f4-786a19ce9e78.png)

**Comment by Fxbouffant** [comment]
Wow now this is weird.

I incorporated your view to my solution and it worked
I created a View and pasted your code into it and it worked
I placed your code in my View and it didn't work

It could only be an encoding issue of the file so I checked  with Notepad++
The old View is in ANSI and the new one in UTF-8-BOM

It seems like, with Visual Studio 2015 Update 3, when you create a View from Add New Item (green), the encoding is UTF-8 but when you create a View from Add View (red), the encoding is not UTF-8.

![image](https://cloud.githubusercontent.com/assets/3736111/20605360/524068ae-b26b-11e6-87a0-8b507ae48f4d.png)

Well I'm sorry I have used your time for something that didn't concern your project and thank you very much for your help.
